audit.py

```diff
@@ -30,6 +30,7 @@
     # unmaintained repos.
     ("softdevteam", "k2"),
     ("softdevteam", "error_recovery_experiment"),
+    ("softdevteam", "WLambda"),
```

Comment:
Is WLambda unmaintained then?

Reply:
For us it's a benchmarking target, so we'll either get those changes from upstream or... not.

Reply:
I think we should add a new section comment like:

```
# maintained externally
```

or similar.

Reply:
Good idea. Fixed here ad7ff8eea4745d5fb6adb7b6579929f01e42710f